assets/app/scripts/controllers/build.js

```diff
@@ -120,10 +129,16 @@ angular.module('openshiftConsole')
               delete $scope.buildConfigBuildsInProgress[buildConfigName][buildName];
             }
           }
-
-          $scope.canBuild = !hashSize($scope.buildConfigBuildsInProgress[buildConfigName]);
         }));
 
+        $scope.isPaused = function() {
+          return BuildsService.isPaused($scope.buildConfig);
+        };
```

Comment:
Better to just set 

```
$scope.paused = BuildsService.isPaused(buildConfig);
```

from the get/watch. Then you only check the annotation when the BC changes, not every Angular digest loop.